src/simulated_bifurcation/models/knapsack.py

```diff
@@ -34,18 +60,26 @@ def content(self) -> Dict[str, Union[int, float, List[int]]]:
             content["items"] = np.arange(1, self.n_items + 1)[items == 1].tolist()
```

Comment:
probably should use zero-based indexing instead of one-based indexing (unless there is a specific reason I'm not aware of)
```suggestion
            content["items"] = np.arange(self.n_items)[items == 1].tolist()
```